source/ui/PGM/PGMPersistentView.h

```diff
@@ -0,0 +1,61 @@
+#ifndef PGMPERSISTENTVIEW_H
+#define PGMPERSISTENTVIEW_H
+
+#include <QWidget>
+
+class QLabel;
+
+class PGMPersistentView : public QWidget
+{
+    Q_OBJECT
+public:
+    explicit PGMPersistentView(QWidget *parent = 0);
+
+    /// Storing the average (x, y) values for the beam position for 10ID, 11ID-1 and 11ID-2
+    /// When the bpm changes, they are allowed to change within +/- 50 or a flag will be raised (color change in text).
+    int avg10IDx_, avg10IDy_, avg11ID1x_, avg11ID1y_, avg11ID2x_, avg11ID2y_;
+    int bpmVariance_;
```

Comment:
I notice that they represent constants, so I would suggest either making them const ints or using #defines
Either way they should be protected instead of public